aws/terminator/data_services.py

```diff
@@ -178,6 +178,23 @@ def terminate(self):
         self.client.delete_db_parameter_group(DBParameterGroupName=self.name)
 
 
+class RdsDBInstance(DbTerminator):
+    @staticmethod
+    def create(credentials):
+        return Terminator._create(credentials, RdsDBInstance, 'rds', lambda client: client.describe_db_instances()['DBInstances'])
+
+    @property
+    def id(self):
+        return self.instance['DBInstanceArn']
+
+    @property
+    def name(self):
+        return self.instance['DBInstanceIdentifier']
+
```

Comment:
```suggestion
    @property
    def age_limit(self):
        return datetime.timedelta(minutes=60)
```
The terminator deletes resources after 20 minutes by default, we can override that with this property though.  The class is definitely working though!  (I killed a few databases in the middle of a test run by forgetting about the 20 minute limit  :))

Reply:
:D yeah that probably isn't ideal !